### ISSUE TITLE
Installation manual

### ISSUE THREAD

**Issue by djkrose** [djkrose/7DTD-ScriptingMod#11]
The mod needs a short installation manual

**Comment by alexmorleyfinch** [comment]
I agree, I tried to install this mod and failed. The documentation on the internet for installing 7dtd mods in general is pretty poor. From what I read on 7dtd forums. you just need to copy the mod into the `Mods` folder, and that's it.

I cloned your repo on my server (in some random place), and copied the `ScriptingMod` folder into the `Mods` folder on my sdtdserver so that it looks like `Mods/ScriptingMod/ModInfo.xml`

I restarted my sdtdserver, telnet'd in and tried to run `dj-version`, but it said the command didn't exist. Is there another obvious step I am missing?
